src/Conversion/KrnlToLLVM/KrnlEntryPoint.cpp

```diff
@@ -89,11 +89,7 @@ class KrnlEntryPointOpLowering : public OpRewritePattern<KrnlEntryPointOp> {
             .getLeafReference()
             .getValue();
 
-    // When there is only a single entry point function in a model, use
-    // DEFAULT_DYN_ENTRY_POINT.
     std::string dynEntryPointName = "run_" + staticEntryPointFuncName.str();
-    if (singleEntryPoint)
-      dynEntryPointName = DEFAULT_DYN_ENTRY_POINT;
```

Comment:
@kernhanda because we don't use `singleEntryPoint` anymore, could you please remove it elsewhere, e.g. in function arguments? Thanks!